ui/client/http/HttpService.ts

```diff
@@ -70,13 +76,12 @@ class HttpService {
       .catch(error => this.addError("Cannot fetch state", error))
   }
 
-  fetchHealthCheckProcessDeployment() {
+  fetchHealthCheckProcessDeployment(): Promise<HealthCheckResponse> {
     return api.get("/app/healthCheck/process/deployment")
       .then(() => ({state: "ok"}))
       .catch((error) => {
-        const errorObj: HealthCheckProcessDeploymentType = error.response?.data
-        const errorMessage: string = errorObj?.processes ? `${errorObj.message}: ${errorObj.processes.join(",")}` : errorObj?.message
-        return {state: "error", error: errorMessage}
+        const {message, processes}: HealthCheckProcessDeploymentType = error.response?.data
+        return {state: "error", error: message, processes: processes}
```

Comment:
Maybe better way is do this state as ENUM? And use here predefined values? I mean instead of state: "error" use  state: STATE.Error.